mobile-app/pubspec.lock

```diff
@@ -970,6 +984,13 @@ packages:
       url: "https://pub.dartlang.org"
     source: hosted
     version: "0.8.15"
+  state_notifier:
```

Comment:
we already use stacked as our provider and state notifier so this is not necessary.

Reply:
ok

Reply:
no problem i will remove it